runtime/server/table_api_test.go

```diff
@@ -0,0 +1,111 @@
+package server
+
+import (
+	"context"
+	"testing"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/stretchr/testify/require"
+
+	"github.com/rilldata/rill/runtime/api"
+	"github.com/rilldata/rill/runtime/drivers"
+)
+
+func getSingleValue(rows *sqlx.Rows) int {
+	var val int
+	for rows.Next() {
+		if err := rows.Scan(&val); err != nil {
+			panic(err)
+		}
+	}
+	rows.Close()
+	return val
+}
+
+func TestServer_Database(t *testing.T) {
+	server, instanceId, err := getTestServer()
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
Can use `require.NoError(t, err)` for these kinds of checks

Reply:
changed